### PR TITLE
Remove GitHub docs in favor of organization level configuration

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Add a issue template config.yml since discussions link is project specific
- Add documentation on use of passwords
- Fix a line rendering error in deployment diagram
- Remove GitHub docs in favor of [organization](https://github.com/hashgraph/.github) level configuration

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
CONTRIBUTING.md [PR](https://github.com/hashgraph/.github/pull/1) still up for review.

**Checklist**

- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -3,10 +3,18 @@
 The four components of the Hedera Mirror Node (Importer, Monitor, REST API, and gRPC API) all support loading
 configuration from an `application.yml` file or via the environment.
 
-Most configuration settings have appropriate defaults and can be left unchanged. One of the important settings that
-should be changed is `hedera.mirror.importer.network` as it controls which of the Hedera networks to mirror.
-Additionally, the password properties have a default, but it is **strongly recommended passwords be changed from the
-default**.
+## Default Values
+
+The default configuration allows users to get quickly up and running without having to configure anything. This provides
```
Comment (Nana-EC) [inline docs/configuration.md:8]: nit: reads better I think
```suggestion
The default configuration allows users to quickly get up and running without having to configure anything. This provides
```

Comment (steven-sheehy) [reply docs/configuration.md]: Updated

**Review by Nana-EC — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Comment by steven-sheehy** [comment]
I ended up being able to add the link to HIP in the organization level config. So no changes necessary here for that.

**Pull request merged**